src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,70 @@
-import React from 'react';
+import { FC, useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
 import { Loader } from '../Loader';
+import { actions } from '../../features/currentTodo';
+import { User } from '../../types/User';
+import { useAppSelector } from '../../app/hooks';
+import { getUser } from '../../api';
+
+export const TodoModal: FC = () => {
+  const todo = useAppSelector(state => state.currentTodo);
+  const dispatch = useDispatch();
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const currUser = async () => {
+      if (todo) {
+        const data = await getUser(todo.userId);
+
+        setUser(data);
+        setIsLoading(false);
+      }
+    };
+
+    currUser();
+  }, []);
```

Comment:
The useEffect dependency array is empty, which means it will only run once on the initial render. However, it should also run whenever the `todo` changes. Add `todo` to the dependency array.